libc/CMakeLists.txt

```diff
@@ -663,13 +663,10 @@ add_library(oelibc STATIC
     ${MUSLSRC}/string/index.c
     ${MUSLSRC}/string/memccpy.c
     ${MUSLSRC}/string/memchr.c
-    ${MUSLSRC}/string/memcmp.c
-    ${MUSLSRC}/string/memcpy.c
     ${MUSLSRC}/string/memmem.c
-    ${MUSLSRC}/string/memmove.c
     ${MUSLSRC}/string/mempcpy.c
     ${MUSLSRC}/string/memrchr.c
-    ${MUSLSRC}/string/memset.c
+    #${MUSLSRC}/string/memset.c
```

Comment:
Commented out files look wrong, remove the line if it's not needed

Reply:
Fixed. Good catch.